File: CR Tejavardhan Reddy/March_2023/08-03-2023/2.py
# Given an array A[] of N positive integers which can contain integers from 1 to P where elements can be repeated or can be absent from the array. Your task is to count the frequency of all elements from 1 to N.
arr=["aaa","bbb","ccc","bbb","aaa","aaa"]
d={}
for ch in arr:
	d[ch]=d.get(ch,0)+1
dict(sorted(d.items(), key=lambda item: item[1]))
print(d)
l=[]
for k in d.keys():
	l.append(k)
if len(l)>1:
	print(l[1])
else:
	print(l[0])

File: CR Tejavardhan Reddy/March_2023/08-03-2023/2.go
package main

import "fmt"

func main() {
	var arr = []string{"aaa", "bbb", "ccc", "bbb", "aaa", "aaa"}
	var list = make([]int, 0)
	var dict = make(map[string]int)
	for _, num := range arr {
		dict[num] += 1
	}
	fmt.Print(dict)
	for _, v := range dict {
		list = append(list, v)
	}

}
